stable/cockroachdb/templates/job.init.yaml

```
@@ -30,16 +30,16 @@ spec:
     spec:
       restartPolicy: OnFailure
       terminationGracePeriodSeconds: 0
-    {{- if or .Values.image.credentials (and .Values.tls.enabled .Values.tls.init.image.credentials) }}
+    {{- if or .Values.image.credentials (and .Values.tls.enabled .Values.tls.init.image.credentials) (not .Values.tls.certs.provided) }}
```

This
```suggestion
    {{- if or .Values.image.credentials (and .Values.tls.enabled .Values.tls.init.image.credentials  (not .Values.tls.certs.provided)) }}
```
doesn't have an effect on the rendered outcome, but keeps the logic more consistent IMHO.